index.js

```diff
@@ -208,11 +208,14 @@ const artists = [
 (1) Name of the first artist (0th index) in the array
 (2) Bio of the third artist (2nd index) in the array */
 
+console.log(artists[0].name)
```

Comment:
You'll be a rockstar & get a headstart if you label your console logs:
console.log("Label in quotes", whateverYourLoggingHere)
`console.log("Task1:", artists[0].name)`